airbyte-integrations/connectors/source-google-workspace-admin-reports/source_google_workspace_admin_reports/client.py

```diff
@@ -41,11 +41,17 @@ def __init__(self, credentials_json: str, email: str, lookback: Optional[int] =
         }
         super().__init__()
 
+    def stream_has_state(self, name: str) -> bool:
+        """Tell if stream supports incremental sync"""
+        return hasattr(self._apis[name], "state")
```

Comment:
seems more robust to check `isinstance(IncrementalStream)` rather than checking if there is a field with a particular name?